### PR TITLE
Request access token

### PR DESCRIPTION
**Pull request by CreedTech**
Added access token to header in fetch-hook for making requests to the BE

### CODE REVIEW

File: src/components/Notification.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+interface INotificationProps {
+    socket: any;
+};
+
+
+const Notification: React.FC<INotificationProps> = ({ socket }) => {
+    const classes = useStyles();
+    const [notifications, setNotifications] = useState<string[]>([]);
+    const [open, setOpen] = useState<any>(false);
```
Comment (pablo-clueless) [inline src/components/Notification.tsx:18]: Type should be <boolean> here not <any>

**Review by pablo-clueless — COMMENTED** [review]
Check for comments in Notifications.

**Comment by taiwoAkindele** [comment]
Pls resolve the conficts on your branch

**Comment by Vinyl-Davyl** [comment]
There are some conflicts on this branch pls check it out

**Pull request closed without merging**